server/homebrew.api.js

```diff
@@ -43,7 +43,9 @@ const getBrew = (accessType, stubOnly = false)=>{
 				}
 			});
 		stub = stub?.toObject();
-		if(accessType === 'edit' && stub?.authors?.length > 0 && !stub?.authors.includes(req.account?.username)) {
+		const authorsExistAndIsNotAuthor = stub?.authors?.length > 0 && !stub?.authors.includes(req.account?.username);
+		const isNotInvited = stub?.invitedAuthors?.length > 0 && !stub?.invitedAuthors.includes(req.account?.username);
```

Comment:
If `stub` doesn't exist, or `invitedAuthors` doesn't exist or is size 0, `isNotInvited` will be FALSE and the user will be allowed access.

The negative variable name is also making the logic tricky to follow (at least for me). Can we make this `isInvited`?

.... actually.... can we change all of these names?

Change `authorsExistAndIsNotAuthor` into `authorsExist` and `isAuthor`, and `isNotInvited` into `isInvited`?

Then the access blocking logic becomes `if(accessType === 'edit' && (authorsExist && !isAuthor || i!sInvited))` or whatever the parentheses need to be.